src/test/groovy/org/arl/fjage/test/spec/fjageSpec.js

```diff
@@ -414,6 +415,117 @@ describe('Shell GetFile/PutFile', function () {
       });
     });
   });
+
+  it('should be able to append a file using PutFileReq', function (done) {
```

Comment:
If you don't provide an `offset`, then it'll overwrite the file right? So is that considered appending? Or overwriting?

Reply:
Yes. it will be overwriting. I will update the description.